packages/code-utils/src/index.ts

```diff
@@ -1,53 +1,72 @@
 import parseOpcode from "./opcodes";
-import type { Instruction, OpcodeTable, opcodeObject, opcodes } from "./types";
-export type { Instruction, OpcodeTable, opcodeObject, opcodes };
+import type { Instruction, OpcodeTable, DisassemblyOptions } from "./types";
+export type { Instruction, OpcodeTable, DisassemblyOptions };
 import * as cbor from "cbor";
```

Comment:
My guess is that this (importing the entire package contents) was the source of the bundle bloat issue that you observed, though I'm betting that `cbor` isn't a TS module, so the TS compiler probably can't do proper tree-shaking here.

Perhaps it's possible that webpack can, but I also doubt it.

Reply:
Hm yeah my understanding is that Webpack's tree-shaking capabilities are pretty limited. :-/  But I also don't know this stuff very well so I could be wrong?

Reply:
I'm in a similar boat. I've had to acquire deep knowledge of what webpack can and can't do in that area, and then I quickly lose it the moment I get away from the problem. From what I remember it has a bunch of facilities for manually pruning out code trees in cases like this, but in doing so you have to be quite confident that you'll never branch into said pruned code. Rather ugly, IMO.